sdk/zksend/src/links/list-created-links.ts

```diff
@@ -16,7 +16,7 @@ const ListCreatedLinksQuery = graphql(`
 		transactionBlocks(
 			last: 10
 			before: $cursor
-			filter: { sentAddress: $address, function: $function, kind: PROGRAMMABLE_TX }
+			filter: { sentAddress: $address, function: $function }
```

Comment:
The kind filter now causes the GraphQL query to fail, but no address should actually have other kinds of transactions, so it wasn't necessary in the first place